### ISSUE TITLE
app registration for national cloud deployments

### ISSUE THREAD

**Issue by DerGuteWolf** [microsoftgraph/microsoft-graph-docs#2007]
Article: https://developer.microsoft.com/en-us/graph/docs/concepts/deployments

This article mentions the service root endpoints and the Azure AD OpenID Connect and OAuth2.0 endpoints.

But what URL/endpoint should one use to register an application for a national cloud deployment? 

For Microsoft Graph global service this is https://apps.dev.microsoft.com/ according to https://developer.microsoft.com/en-us/graph/docs/concepts/auth_register_app_v2 .

There is nothing mentioned regarding this on https://developer.microsoft.com/en-us/graph/docs/concepts/deployments

**Comment by marcreicher** [comment]
@jthake-msft @DerGuteWolf I have a similar question. I have a customer in the US Gov cloud. It seems like my multi-tenant app that I have registered in AAD (at `portal.azure.com`) is unable to acquire tokens on behalf of a US Gov cloud tenant. I can't tell if this is by design or not. Is the idea that I have to procure an Azure US Gov tenant and register an application in that tenant (at `portal.azure.us`) in order to integrate with other US Gov Cloud tenants?

**Comment by DerGuteWolf** [comment]
@marcreicher IMHO the registration URL has moved from https://apps.dev.microsoft.com/ to https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade .
So for US Gov this should be https://portal.azure.us/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade which I obviously can't access because I am lacking such an account/tenant and thus can't verifiy,

**Comment by marcreicher** [comment]
@DerGuteWolf Yeah it's frustrating this page is not clearer. I believe you are correct though. Unfortunately, it is much easier said than done to get access to `portal.azure.us`!
